frontend/src/augmented.css

```diff
@@ -0,0 +1,1556 @@
+/*
```

Comment:
https://libraries.io/npm/augmented-ui
Nit: Could make it a dependency instead of including the stylesheet in the repo ;)

Reply:
I could, but I didn't want to. I felt like if something changed in this beautiful magic, it could break.